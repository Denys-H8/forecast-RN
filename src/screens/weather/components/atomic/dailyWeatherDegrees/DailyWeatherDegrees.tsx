import React from 'react';
import { View } from 'react-native';

import CustomText from 'src/components/text';
import { styles } from './dailyWeatherDegrees.styles';

interface IProps {
  min: number;
  max: number;
}

const DailyWeatherDegrees: React.FC<IProps> = ({ min, max }) => (
  <View>
    <CustomText style={styles.number}>{`${min} /`}</CustomText>
    <View style={styles.container}>
      <CustomText style={styles.number}>{max}</CustomText>
      <CustomText style={styles.celsius}>0</CustomText>
      <CustomText style={styles.number}>C</CustomText>
    </View>
  </View>
);

export default DailyWeatherDegrees;
