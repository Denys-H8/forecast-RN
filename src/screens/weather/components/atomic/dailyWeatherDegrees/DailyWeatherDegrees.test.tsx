import React from 'react';
import { render } from '@testing-library/react-native';
import DailyWeatherDegrees from './DailyWeatherDegrees';

const min = 15;
const max = 27;

describe('DailyWeatherDegrees ', () => {
  test('should render DailyWeatherDegrees', () => {
    const { toJSON } = render(<DailyWeatherDegrees min={min} max={max} />);

    expect(toJSON()).toMatchSnapshot();
  });
});
