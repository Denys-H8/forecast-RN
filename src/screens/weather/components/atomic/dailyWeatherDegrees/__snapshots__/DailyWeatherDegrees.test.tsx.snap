// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DailyWeatherDegrees  should render DailyWeatherDegrees 1`] = `
<View>
  <Text
    style={
      Array [
        Object {
          "fontFamily": "WorkSans-Regular",
        },
        Object {
          "color": "#fff",
          "fontSize": 18,
          "lineHeight": 20,
        },
      ]
    }
  >
    15 /
  </Text>
  <View
    style={
      Object {
        "alignItems": "flex-start",
        "flexDirection": "row",
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "fontFamily": "WorkSans-Regular",
          },
          Object {
            "color": "#fff",
            "fontSize": 18,
            "lineHeight": 20,
          },
        ]
      }
    >
      27
    </Text>
    <Text
      style={
        Array [
          Object {
            "fontFamily": "WorkSans-Regular",
          },
          Object {
            "color": "#fff",
            "fontSize": 12,
            "lineHeight": 12,
          },
        ]
      }
    >
      0
    </Text>
    <Text
      style={
        Array [
          Object {
            "fontFamily": "WorkSans-Regular",
          },
          Object {
            "color": "#fff",
            "fontSize": 18,
            "lineHeight": 20,
          },
        ]
      }
    >
      C
    </Text>
  </View>
</View>
`;
